wbooks/src/app/screens/booklist/components/BookItem/index.js

```diff
@@ -1,21 +1,30 @@
 import React, { Component } from 'react';
 import { View, Image, Text } from 'react-native';
 import styles from './styles';
+import PropTypes from 'prop-types';
 
 class BookItem extends Component {
 
-	render() {
-		return (
+  render() {
+    const { uri, title, writer } = this.props;
+    
+    return (
       <View style={styles.container}>
         <Image style={styles.image}
-        source={{uri: 'http://wolox-training.s3.amazonaws.com/uploads/6942334-M.jpg'}} />
+        source={{uri}} />
         <View style={styles.textContainer}>
-          <Text style={styles.bookTitle}>A Little Bird Told Me</Text>    
-          <Text style={styles.bookDescription}>Timothy Cross</Text>    
+          <Text style={styles.bookTitle}>{title}</Text>    
+          <Text style={styles.bookDescription}>{writer}</Text>    
```

Comment:
i think `author` is more appropriate, but `writer` is fine too.

Reply:
you're right, changed